source/includes/steps-starting-vsce-paste-string.yaml

```diff
@@ -0,0 +1,63 @@
+---
+title: Obtain your connection string.
+level: 4
+ref: vsce-enter-connection-info
+content: |
+ 
+  You can use either the :manual:`Standard Connection String Format
+  </reference/connection-string/#connections-standard-connection-string-format>`
+  or the :manual:`DNS Seedlist Connection Format
+  </reference/connection-string/#connections-dns-seedlist>`.
+
+  - To obtain the connection string for an |service| cluster:
+  
+    a. Navigate to your |service| :guilabel:`Clusters` view.
+
+    #. Click :guilabel:`Connect` for your desired cluster.
+
+    #. Click :guilabel:`Connect with MongoDB Compass`.
+
+    #. Copy the provided connection string.
+
+  - To learn how to format the connection string for a deployment
+    which is not hosted on |service|, see
+    :manual:`Connection String URI Format
+    </reference/connection-string>`.
+---
+source:
+  file: steps-source-command-palette.yaml
+  ref: vsce-command-palette
+ref: vsce-connect-command-palette
+---
+title: "From the Command Palette, select 
+      :guilabel:`MongoDB: Connect with Connection String...`."
+level: 4
+ref: vsce-connect-connect
+content: |
+
+  .. include:: /includes/admonitions/tip-mongodb-command-palette.rst
+
+---
+
+title: "Paste your connection string into the Command Palette."
+level: 4
+ref: vsce-connect-paste
+content: |
+
+  .. important::
+
+     Ensure that you update the connection string with credentials
+     of a database user for your deployment. The connection might 
+     succeed without these credentials, but you won't be able to
+     interact with data in your deployment.
```

Comment:
This block is confusing to me. What if I'm running without authentication? (shame on me, but lets say I'm just testing locally). Should we reframe this to say "If your deployment requires authentication, ensure that you update the connection string with credentials of a database user for your deployment."

Reply:
updated